rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2397,6 +2397,40 @@ void voteFederationChange_addFederatorPublicKeyWithLessThanMofNVotes_returnsSucc
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorPublicKey_returnsSuccessfulResponseCode() {
+
+            // Arrange
+
+            BtcECKey expectedBtcECKey = new BtcECKey();
+
+            Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
```

Comment:
```suggestion
            Transaction firstAuthorizedTx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
```

I would use a more descriptive name for this variable and the one below. Wdyt?

Reply:
Updated.